### ISSUE TITLE
False warning about missing key with BiBTeX @strings

### ISSUE THREAD

**Issue by bersbersbers** [plk/biber#345]
While the output is as expected, this issues a warning about a missing key:

```
echo "@string{TEST={test}}" > test.bib
echo "@article{ART,title=TEST}" >> test.bib

biber --tool test.bib -O -
```

```
INFO - This is Biber 2.16 (beta) running in TOOL mode
INFO - Logfile is 'test.bib.blg'
INFO - Looking for bibtex format file 'test.bib'
INFO - LaTeX decoding ...
INFO - Found BibTeX data source 'test.bib'
WARN - Invalid or undefined BibTeX entry key in file '/tmp/biber_tmp_j8Vr/5a453152acdb3e2485a9d6a17b3ec69c_8289.utf8', skipping ...
INFO - Overriding locale 'en_US' defaults 'normalization = NFD' with 'normalization = prenormalized'
INFO - Overriding locale 'en_US' defaults 'variable = shifted' with 'variable = non-ignorable'
INFO - Sorting list 'tool/global//global/global' of type 'entry' with template 'tool' and locale 'en_US'
INFO - No sort tailoring available for locale 'en_US'
INFO - Writing '-' with encoding 'UTF-8'
@STRING{TEST = "test"}

@ARTICLE{ART,
  TITLE = TEST,
}

INFO - Output to -
INFO - WARNINGS: 1
```

**Comment by plk** [comment]
Ah, yes, this was a bug, should be fixed in 2.16 DEV now.

**Comment by bersbersbers** [comment]
Looks good!
